src/pages/STG_PanelCustomizableRollup.page

```diff
@@ -61,52 +59,118 @@
                     </div>
                 </div>
             </apex:outputPanel>
+            <!-- BEGIN NON-ADMIN MESSAGE -->
+
+            <!-- BEGIN INTRO TEXT AND TOGGLE -->
+            <div class="slds-grid {!IF(isHHAccount, '', 'slds-hide')}">
+                <apex:outputPanel rendered="{! !isPolling }">
+                    <div class="slds-m-vertical_medium">
+                        <apex:outputText escape="false" styleClass="slds-text-body_small" value="{!$Label.stgHelpCustomizableRollupsEnable1}"/>
+                    </div>
+
+                    <label class="slds-checkbox_toggle slds-grid slds-m-vertical_small">
+                        <span class="slds-form-element__label slds-m-bottom_none">Customizable Rollups</span>
+                        <apex:inputCheckbox onClick="onToggle(); return false;" id="enableCRLPToggle" html-name="checkbox-toggle-2" value="{!stgService.stgCRLP.Customizable_Rollups_Enabled__c}" html-aria-describedby="checkbox-toggle-2" />
```

Comment:
This checkbox isn't keyboard accessible when CRLPs is off, and when it's on it's actually navigating to the app. This looks like a problem with the apex:inputCheckbox, but maybe is a result of using onClick vs using onSelect?

Reply:
This is working with space! Looks great.